cypress/support/step_definitions/carouselSteps.js

```diff
@@ -0,0 +1,105 @@
+import {
+  slide, slideIndexSelect, carousel, nextArrowButton, previousArrowButton, slideSelectorIndex,
+  enableSlideSelectorCheckbox, slideSelector, enableNextButtonCheckbox, enablePreviousButtonCheckbox, transitionSelect, giveTransition
+} from '../../locators/carouselLocators';
+import { DEBUG_FLAG } from '..';
+
+function waitForCarouselMove() {
+  carousel().should('have.length', 2, { log: DEBUG_FLAG }); // two slides visible during carousel move 
+  carousel().should('have.length', 1, { log: DEBUG_FLAG }); // one slide visible after carousel move
+}
+
+function clickCarouselButton(direction) {
+  switch (direction) {
```

Comment:
I think these could be inline.

Reply:
I think that we'll lose readability when they will be inline

Reply:
I'm just conscious that this file is already long and doesn't seem to have organisation.

Reply:
It would only reduce the lines count from 105 to 101.